CodeFirst/Assement.cs

```diff
@@ -0,0 +1,136 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace TechnicalAssesment
+{
+    public enum GameResult
+    {
+        GameNotStarted,
+        DealerWon,
+        PlayerWon,
+        Tie,
+    }
+
+    public class ScoreBoard
+    {
+        public byte playerWins;
+        public int DealerWins;
+        public byte Ties { get; set; }
+    }
+
+    public class Deck
+    {
+        public List<int> cards;
+        public int MaxCards = 52;
+        public Deck()
+        {
+            cards = new List<int>();
```

Comment:
Use of assigning value to Cards can can be done on one line. EG public List<int> cards; = new List<int>(); Wasted used of a constructor as its being used inside the deck class